hub/src/server/utils.ts

```diff
@@ -43,6 +43,14 @@ export function getDriverClass(driver: DriverName): DriverConstructor & DriverSt
   }
 }
 
+export function megabytesToBytes(megabytes: number) {
+  return megabytes * 1024 * 1024
+}
+
+export function bytesToMegabytes(bytes: number, decimals = 2) {
+  return Number.parseFloat((bytes / 1024 / 1024).toFixed(decimals))
```

Comment:
Not a bug, but I'd prefer `bytes / (1024 * 1024)` -- I think `/` is always left-binding, but this would play it safe.

Reply:
Fixed